tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -718,17 +730,22 @@ def test_positive_rhev_vm_power_on(self):
             ['Password', self.rhev_password, 'field'],
             ['Datacenter', self.rhev_datacenter, 'special select'],
         ]
-        name = gen_string('alpha')
+        cr_name = gen_string('alpha')
         with Session(self.browser) as session:
             make_resource(
                 session,
-                name=name,
+                name=cr_name,
                 provider_type=FOREMAN_PROVIDERS['rhev'],
                 parameter_list=parameter_list
             )
+            if self.compute_resource.power_on_status(
```

Comment:
With these fixes you are making test cases almost identical in terms of functionality. If it is not possible to know the state in advance, so i think it is better to combine these tests into one